streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -326,6 +326,100 @@ void print(final Serde<K> keySerde,
                final String streamName);
 
     /**
+     * Print the customized output with {@code System.out}.
+     * <p>
+     * The default serde will be use to deserialize key or value if type is {@code byte[]}.
+     * The user provided {@link KeyValueMapper} which customizes output is used to print with {@code System.out}
+     * <p>
+     * The example below shows the way to customize output data.
```

Comment:
@bbejeck I think no need to mention this due to only accept String type in KeyValueMapper's output part.